src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -495,7 +495,9 @@ public static void onClientTick(@NotNull TickEvent.ClientTickEvent event)
     public static void onWorldTick(
                                     @NotNull TickEvent.WorldTickEvent event)
     {
-        colonies.values().stream()
+        Map<Integer, Colony> coloniesCopy = new HashMap<>(colonies);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'coloniesCopy' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)